.github/workflows/pythonapp.yml

```diff
@@ -29,12 +29,14 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
+        pip install pytest
```

Comment:
I think we should just add pytest to requirements.txt instead of having the separate line, but I'm eager enough to get this merged (before anyone write new tests we'd have to convert) so I think I'll just make a separate PR for that.